include/common/mir/time/posix_clock.h

```diff
@@ -0,0 +1,51 @@
+
+#ifndef MIR_COMMON_TIME_POSIX_CLOCK_
+#define MIR_COMMON_TIME_POSIX_CLOCK_
+
+#include <chrono>
+
+namespace mir::time
+{
+template<clockid_t clock_id>
+struct is_steady_specialisation
+{
+    static constexpr bool is_steady = false;
+};
+
+template<>
+struct is_steady_specialisation<CLOCK_MONOTONIC>
```

Comment:
I think there's a missing `<time.h>`/`<ctime>` header: `CLOCK_MONOTONIC` might be transitively defined by `<chrono>`, but that's not part of the standard